contracts/l1/Bridge.sol

```diff
@@ -140,16 +156,16 @@ contract Bridge is IBridge, VersionedInitializable {
         address recipient,
         uint256 staticAmount,
         uint256 l2RewardsIndex,
-        bool toUnderlyingAsset
+        uint256 toUnderlyingAsset
```

Comment:
tiny: `uint8` is enough here

Reply:
let's stick with uint256 for better packing

Reply:
There is no packing on memory and calldata, only on storage

Reply:
Also, it is quite inconsistent to have `bool` on deposit and `uintX` on withdrawal.
Even if I assume it is for convenience on the bridge side on withdrawal, better to unify 1 way or another the interface